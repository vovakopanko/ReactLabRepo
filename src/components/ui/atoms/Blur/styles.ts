import styled from "styled-components";

export const BlurEffect = styled.div`
  width: 100%;
  padding: 20px;
  margin-top: 20px;
  backdrop-filter: blur(2px) grayscale(0.5);
  border-radius: 25px;
`;
