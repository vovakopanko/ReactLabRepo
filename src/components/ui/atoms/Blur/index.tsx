import { FC, ReactNode } from "react";
import { BlurEffect } from "./styles";

type Props = {
  children?: ReactNode;
};

const Blur: FC<Props> = ({ children }) => {
  return <BlurEffect>{children}</BlurEffect>;
};

export default Blur;
