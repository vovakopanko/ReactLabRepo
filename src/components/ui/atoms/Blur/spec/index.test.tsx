import renderer from "react-test-renderer";
import Blur from "../index";

describe("Blur component unit test", () => {
  it("should return Blur layout component", () => {
    const tree = renderer.create(<Blur />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
