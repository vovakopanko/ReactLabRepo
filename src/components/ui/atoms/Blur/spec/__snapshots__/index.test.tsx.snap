// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blur component unit test should return Blur layout component 1`] = `
<div
  className="sc-bdvvtL eLBDUL"
/>
`;
